scipy/conftest.py

```diff
@@ -5,12 +5,16 @@
 import pytest
 import warnings
 
+from distutils.version import LooseVersion
 from scipy._lib._fpumode import get_fpu_mode
 from scipy._lib._testutils import FPUModeChangeWarning
 
 
 def pytest_runtest_setup(item):
-    mark = item.get_marker("xslow")
+    if LooseVersion(pytest.__version__) >= LooseVersion("3.6.0"):
```

Comment:
i just remembered that `xslow in item.keywords` might actually work (i wont be able to test this out soon)